rockcraft.yaml

```diff
@@ -41,5 +42,18 @@ parts:
             chown -R 584788 $CRAFT_PRIME/var/log/pgbackrest
             chown -R 584788 $CRAFT_PRIME/var/log/postgresql
             chown -R 584788 $CRAFT_PRIME/var/log/prometheus
+            mkdir -p $CRAFT_PRIME/var/log/pgbouncer
+            chown -R 584788 $CRAFT_PRIME/var/log/pgbouncer
             mkdir -p $CRAFT_PRIME/var/run/postgresql
             chown -R 584788 $CRAFT_PRIME/var/run/postgresql
+            mkdir -p $CRAFT_PRIME/var/lib/pgbouncer
+            chown -R 584788 $CRAFT_PRIME/var/lib/pgbouncer
+            # Generate dpkg.query
+            mkdir -p $CRAFT_PRIME/usr/share/rocks
+            echo "# os-release" > $CRAFT_PRIME/usr/share/rocks/dpkg.query
+            cat ${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/etc/os-release >> $CRAFT_PRIME/usr/share/rocks/dpkg.query
+            echo "# dpkg-query" >> $CRAFT_PRIME/usr/share/rocks/dpkg.query
+            dpkg-query -f \
+            '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W \
+            --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ \
+            >> $CRAFT_PRIME/usr/share/rocks/dpkg.query
```

Comment:
Copied the base image location from a MM chat. The overlay packages are not included here.